inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_LSTMCell.cpp

```diff
@@ -106,9 +106,10 @@ void MultipleLSTMCellTest::SetUp() {
     // TI construction
     auto tensor_iterator = std::make_shared<TensorIterator>();
     tensor_iterator->set_body(body);
-    tensor_iterator->set_invariant_input(X, permute_in);
+    tensor_iterator->set_sliced_input(X, permute_in, 0, 1, 1, -1, 0);
```

Comment:
just to understand, what is a difference between ngraph and legacy transformations which requires this test change? Also this test seems to work for cpu plugin.